src/Stratis.Bitcoin.IntegrationTests/Mempool/MemoryPoolTests.cs

```diff
@@ -447,5 +447,41 @@ public void Mempool_SendPosTransaction_AheadOfFutureDrift_ShouldRejectByMempool(
                 Assert.Equal("time-too-new", entry.ErrorMessage);
             }
         }
+
+        [Fact]
+        public void Mempool_SendOversizeTransaction_ShouldRejectByMempool()
```

Comment:
Note that we do have a test of the mempool validator already that tests for this condition. Is it necessary to introduce an integration test for this?

See: `AcceptToMemoryPool_TxIsNonStandardTransactionSizeInvalid_ReturnsFalseAsync`

Reply:
I'll accept it anyway.